ubuntudesign/developer.ubuntu.com
# Add global nav

## QA
- Run `./run`
- Go through a few pages and check the global nav appears
- Make sure I haven't missed a template

_Not responsive yet and the width is not correct. Awaiting design review to update_

Fixes: https://github.com/ubuntudesign/developer.ubuntu.com/issues/11
